nhrpd/nhrp_interface.c

```diff
@@ -351,6 +351,7 @@ void nhrp_interface_update(struct interface *ifp)
 		if (!if_ad->configured) {
 			os_configure_dmvpn(ifp->ifindex, ifp->name,
 					   afi2family(afi));
+			nhrp_interface_update_arp(ifp, true);
```

Comment:
did you move this out of "configure_dmvpn" on purpose, for some specific reason?

Reply:
the goal is to migrate all things from nhrp_linux.c